Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -957,6 +957,27 @@ final class SyncChangeStreamTests: MongoSwiftTestCase {
 
         // expect this change stream to have more events after resuming
         expect(try changeStream2.nextWithTimeout()).toNot(beNil())
+
+        guard try client.serverVersion() >= ServerVersion(major: 4, minor: 2) else {
+            print("Skipping test case for server version \(try client.serverVersion())")
+            return
+        }
+
+        // test with startAfter
+        _ = try coll.renamed(to: self.getCollectionName(suffix: "2"))
+
+        _ = try changeStream2.nextWithTimeout()
+        expect(try changeStream2.nextWithTimeout()?.operationType).to(equal(.invalidate))
+
+        let opts = ChangeStreamOptions(startAfter: changeStream2.resumeToken)
+
+        // resuming (with startAfter) after an invalidate event should work
+        expect(try coll.watch(options: opts)).toNot(throwError())
```

Comment:
rather than creating multiple change streams here, I think it's okay to just rely on `try` to report any test failures and not do the `expect(...).toNot(throwError())`.